.github/workflows/security.yml

```diff
@@ -0,0 +1,34 @@
+name: Run Gosec
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - '**'
+jobs:
+  tests:
+    runs-on: ubuntu-latest
+    env:
+      GO111MODULE: on
+    steps:
+      - name: Checkout Source
+        uses: actions/checkout@v2.4.0
+      # - uses: technote-space/get-diff-action@v5
+      #   with:
+      #     SUFFIX_FILTER: |
+      #       .go
+      #       .mod
+      #       .sum
+      - name: Run Gosec Security Scanner
+        uses: informalsystems/gosec@master
```

Comment:
@odeke-em I assume we need to use the informal version? do we need to also add a custom rule for the SDK?